server/src/main/java/org/spine3/server/event/enrich/EventEnrichmentsMap.java

```diff
@@ -64,24 +64,43 @@ private EventEnrichmentsMap() {}
     }
 
     private static ImmutableMultimap<TypeName, TypeName> buildEnrichmentsMap() {
-        final ImmutableMultimap.Builder<TypeName, TypeName> builder = ImmutableMultimap.builder();
         final ImmutableSet<Properties> propertiesSet = loadAllProperties(PROPS_FILE_PATH);
-        for (Properties properties : propertiesSet) {
-            putTo(builder, properties);
-        }
+        final Builder builder = new Builder(propertiesSet);
         final ImmutableMultimap<TypeName, TypeName> result = builder.build();
         return result;
     }
 
-    @SuppressWarnings("MethodWithMultipleLoops") // is OK in this case
-    private static void putTo(ImmutableMultimap.Builder<TypeName, TypeName> enrichmentsMap, Properties properties) {
-        final Set<String> enrichments = properties.stringPropertyNames();
-        for (String enrichment : enrichments) {
-            final TypeName enrichmentType = TypeName.of(enrichment);
-            final String eventTypesStr = properties.getProperty(enrichment);
-            final Iterable<String> eventTypes = FluentIterable.of(eventTypesStr.split(EVENT_TYPE_SEPARATOR));
+    private static class Builder {
+
+        private final Set<Properties> propertiesSet;
+        private final ImmutableMultimap.Builder<TypeName, TypeName> builder;
+
+        /* package */ Builder(Set<Properties> propertiesSet) {
+            this.propertiesSet = propertiesSet;
+            this.builder = ImmutableMultimap.builder();
+        }
+
+        /* package */ ImmutableMultimap<TypeName, TypeName> build() {
+            for (Properties properties : propertiesSet) {
+                put(properties);
+            }
+            final ImmutableMultimap<TypeName, TypeName> result = builder.build();
```

Comment:
You can just return `builder.build()`. We have an opportunity to debug/inspect already.